### PR TITLE
[RSS-13] Forgot Password Page

### PR DESCRIPTION
**Pull request by pgonza25**
Created the forgot password page. Includes basic error handling in the case of no email entered.

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...rorCount":0,"fixableWarningCount":0},{"filePath":"21","messages":"22","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"C:\\Users\\pgonz\\Desktop\\rise-stand-strong\\src\\index.js",[],["23","24","25","26","27","28"],"C:\\Users\\pgonz\\Desktop\\rise-stand-strong\\src\\App.js",[],"C:\\Users\\pgonz\\Desktop\\rise-stand-strong\\src\\components\\Main\\Main.jsx",[],"C:\\Users\\pgonz\\Desktop\\rise-stand-strong\\src\\components\\LoginPage\\LoginPage.jsx",[],"C:\\Users\\pgonz\\Desktop\\rise-stand-strong\\src\\components\\Main\\ForgotPassword\\ForgotPassword.jsx",["29"],{"ruleId":"30","replacedBy":"31"},{"ruleId":"32","replacedBy":"33"},{"ruleId":"34","replacedBy":"35"},{"ruleId":"36","replacedBy":"37"},{"ruleId":"38","replacedBy":"39"},{"ruleId":"40","replacedBy":"41"},{"ruleId":"42","severity":1,"message":"43","line":18,"column":7,"nodeType":"44","messageId":"45","endLine":18,"endColumn":37},"lines-around-directive",["46"],"no-spaced-func",["47"],"global-require",[],"no-buffer-constructor",[],"no-new-require",[],"no-path-concat",[],"no-alert","Unexpected alert.","CallExpression","unexpected","padding-line-between-statements","func-call-spacing"]
```
Comment (jackfales) [inline .eslintcache]: Remove this file

File: src/components/Main/Main.jsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
-import LoginPage from '../LoginPage/LoginPage';
+import ForgotPassword from './ForgotPassword/ForgotPassword';
```
Comment (jackfales) [inline src/components/Main/Main.jsx]:
```suggestion
```

File: src/components/Main/Main.jsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
 export default () => (
   <div className="App">
     <Switch>
-      <Route exact path="/" render={() => <LoginPage />} />
+      <Route exact path="/" render={() => <ForgotPassword />} />
```
Comment (jackfales) [inline src/components/Main/Main.jsx]:
```suggestion
```

**Review by jackfales — APPROVED** [review]
Look good, just remove the .eslintcache file and the few lines you changed in `Main.jsx`

**Review by jpoist97 — COMMENTED** [review]
Looks like Jack caught everything already, just fix the stuff he mentioned and you should be good to go

**Review by jackfales — APPROVED** [review]
Looks good

**Pull request merged**